spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher/src/main/java/org/springframework/cloud/kubernetes/configuration/watcher/BusEventBasedConfigMapWatcherChangeDetector.java

```diff
@@ -28,18 +28,16 @@
 import org.springframework.cloud.kubernetes.commons.config.reload.ConfigReloadProperties;
 import org.springframework.cloud.kubernetes.commons.config.reload.ConfigurationUpdateStrategy;
 import org.springframework.context.ApplicationEventPublisher;
-import org.springframework.context.ApplicationEventPublisherAware;
```

Comment:
so I removed `ApplicationEventPublisherAware` and instead autowire `ApplicationEventPublisher` directly. For this specific PR it just makes things cleaner. For the next one, it makes a lot more sense; but imho: even this only reason is enough